proxy/client.go

```diff
@@ -81,6 +91,15 @@ func (c *Client) newRequest(method, path string, body io.Reader) (*http.Request,
 //doRequest perform an HTTP request with context
 func (c *Client) doRequest(ctx context.Context, req *http.Request) (*http.Response, error) {
 	req = req.WithContext(ctx)
+
+	dump, foundDump := os.LookupEnv("OPENFAAS_DUMP_HTTP")
```

Comment:
We can inline these two lines, but not needed to merge.

https://github.com/alexellis/derek/blob/master/auth/customers.go#L21